From 96091811c412ca54767be7f7c6a2e5b6c26ac338 Mon Sep 17 00:00:00 2001
From: "tao.dong" <tao.dong@amlogic.com>
Date: Thu, 9 Oct 2014 11:03:48 +0800
Subject: [PATCH 4927/5965] it cause boot_video problem:Revert "PD
 #96000:enable di auto clock gate."

This reverts commit cae9dd59edc23a0090e21f3eb0ae9557abf17101.
---
 drivers/amlogic/deinterlace/deinterlace_hw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/deinterlace/deinterlace_hw.c b/drivers/amlogic/deinterlace/deinterlace_hw.c
index b6007a7f075e..274a80205ba8 100755
--- a/drivers/amlogic/deinterlace/deinterlace_hw.c
+++ b/drivers/amlogic/deinterlace/deinterlace_hw.c
@@ -331,7 +331,7 @@ void di_hw_init(void)
 {
 #ifdef NEW_DI_V1
     Wr(DI_MTN_1_CTRL1, Rd(DI_MTN_1_CTRL1)&(~(1<<31))); //enable old DI mode for m6tv
-    Wr(DI_CLKG_CTRL, Rd(DI_CLKG_CTRL)); //di clock gate
+    Wr(DI_CLKG_CTRL, Rd(DI_CLKG_CTRL)|0x1); //di no clock gate
 
     /* fifo size setting from 0x1be60 to 0x1bf20 */
     Wr(VD1_IF0_LUMA_FIFO_SIZE, 0x1bf20);  // 1a63 is vd1_if0_luma_fifo_size
-- 
2.19.0

